pinot-plugins/pinot-stream-ingestion/pinot-kafka-2.0/src/main/java/org/apache/pinot/plugin/stream/kafka20/KafkaMessageBatch.java

```diff
@@ -92,6 +92,11 @@ public StreamPartitionMsgOffset getNextStreamPartitionMsgOffsetAtIndex(int index
     return new LongMsgOffset(((KafkaStreamMessage) _messageList.get(index)).getNextOffset());
   }
 
+  @Override
+  public StreamPartitionMsgOffset getStreamPartitionMsgOffsetAtIndex(int index) {
+    return new LongMsgOffset(((KafkaStreamMessage) _messageList.get(index)).getOffset());
```

Comment:
Didn't the default interface method implementation work? I am wondering if we need to implement this method for other stream types as well.

Reply:
For other stream types, I saw it uses the default method. I tried to maintain consistency with what we were doing for `getNextOffset` method.